frontend/src/app/chatmenu/ChatMenuPage.tsx

```diff
@@ -11,7 +11,16 @@ export interface ChatMenuConfig {
     options: Array<ChatMenuPageConfig>
 }
 
+export interface MessagesData {
+    key: string
+    type: string
+    timestamp: number
+    message: string
+    from: string
+}
```

Comment:
Thanks for adding this! I didn't realize we didn't have a Message type definition in the frontend anywhere.

In the future, we might move this to a common location so that other components can use it, but let's keep it here for now.